web/static/css/app.css

```diff
@@ -372,7 +371,7 @@ a {
 .action__button--white {
   border-radius: 3px;
   background-color: white;
-  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.24), 0 0 1px 0 rgba(0, 0, 0, 0.12);
+  box-shadow: 0px 1px 2px 1px rgba(0, 0, 0, .24), 0 0 1px 0 rgba(0, 0, 0, .12);
```

Comment:
Just curious how you decided on this? Just eyeballing it or was it exported from zeplin?